app/pages/[page].js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { applySession } from "next-session";
+import { useRouter } from "next/router";
+import { Forms, getHandler } from "../form-schema";
+import Button from "@button-inc/bcgov-theme/Button";
+
+export default function home({ formIndex, formData, validPage, prevPageUrl }) {
+  const Form = Forms[formIndex];
+  const router = useRouter();
+  const onFirstPage = prevPageUrl === -1;
+
+  const rerouteHandler = (nextPage, _isValid, lastPage) => {
+    router.push(lastPage ? "/end" : nextPage);
+  };
+
+  const handleBackClick = () => {
+    if (onFirstPage) return;
+    router.push(prevPageUrl);
+  };
+
+  return (
+    <>
+      {validPage && (
+        <Form formData={formData} rerouteHandler={rerouteHandler}>
+          {!onFirstPage && (
+            <Button type="button" variant="secondary" onClick={handleBackClick}>
+              Back
+            </Button>
+          )}
+          <Button variant="primary">Continue</Button>
```

Comment:
This is some handling from the `rjsf` base library, if you add a button inside the form it will act as a submit button. I gave the _Back_ button a type `button` to stop it submitting